presets/TestSamples/4.3/race_rc_smoothing.txt

```diff
@@ -0,0 +1,19 @@
+# Title: Race RC smoothing
+# FirmwareVersion: 4.3
+# Category: RC_SMOOTHING
+# Official: true
+# Keywords: aggressive, rc, smoothing, race
+# Author: Chris Thompson ctzsnooze
+# Description: RC smoothing settings for racing - assumes a good RC link
+# Description: Slightly increases stick responsiveness over defaults
+# Description: Note: defaults are close to ideal for racing
+# Description: Warning: sends some noise to the motors at link frequency during fast stick movements - may make motors warmer than usual
+
+set rc_smoothing = ON
+set feedforward_jitter_reduction = 5
+set feedforward_transition = 0
+set rc_smoothing_auto_factor = 22
+set rc_smoothing_auto_factor_throttle = 20
```

Comment:
Let's `# include` defaults file for each preset at the beginning?

For example `rc_smoothing_auto_factor_throttle = 20` is used for race_rc_smoothing.txt
However "cinematic_rc_smoothing.txt" does not set this value at all.

It means that if someone applies "race" and then he wants to try "cinematic", he will have "rc_smoothing_auto_factor_throttle = 20" from the race tune left.

Reply:
I specifically only overwrote the files that needed overwriting.
The cinematic rc smoothing one, once a manual value is set, the user's old auto smoothing value can stay, since it doesn't matter what the auto value is once you use a manual setting..  That's why I did it that way.  
I could set the auto value to a cinematic value but again specifically didn't want to do that.

Reply:
@ctzsnooze it's probably a long conversation, but i think that presets are in general for the people who don't mess with individual parameters. They want to try "race_rc_smoothing", then "cinematic" and see what works best. They don't need to think what smoothing parameters will be left. These people don't need any leftovers between applying/trying these presets.

Reply:
They can do exactly that.  I checked that on each occurrence, what the person writes is what they get.
We don't need to change values that a user may have carefully configured previously, unless we must change them for the preset to work.
If they want to revert to defaults they can run the default first, then the preset.
But I don't want to force them to do so every time.